hazelcast/src/test/java/com/hazelcast/version/VersionUnknownTest.java

```diff
@@ -128,7 +128,7 @@ public void unknown_is_unknownLessOrEqual_unknown() throws Exception {
 
     @Test
     public void any_notEquals_to_unknown() throws Exception {
-        assertNotEquals(ANY_VERSION, UNKNOWN);
+        assertNotEquals(UNKNOWN, ANY_VERSION);
```

Comment:
This one should  be reverted as the original test is the expected one. See the `unknown_notEquals_to_any` for the other direction.
```suggestion
        assertNotEquals(ANY_VERSION, UNKNOWN);
```